tests/kernels/moe/test_deepep_moe.py

```diff
@@ -154,20 +154,23 @@ def make_modular_kernel(pg: ProcessGroup, pgi: ProcessGroupInfo,
                         deepep_ll_args = ll_args)
 
     if low_latency_mode:
+        # TODO(bnell): block_shape?
         fused_experts = BatchedTritonExperts(
             max_num_tokens=MAX_TOKENS_PER_RANK,
             world_size=pgi.world_size,
             dp_size=dp_size,
             use_fp8_w8a8=is_quantized,
             use_int8_w8a8=False,
             use_int8_w8a16=False,
-            use_int4_w4a16=False)
+            use_int4_w4a16=False,
+            per_act_token_quant=False)
     else:
+        # TODO(bnell): block_shape?
```

Comment:
Could you either fix the TODOs before landing or otherwise make them a bit more self-explanatory?